src/bin/ion/commands/inspect.rs

```diff
@@ -214,24 +241,29 @@ impl<'a, 'b> IonInspector<'a, 'b> {
         reader: &mut SystemReader<AnyEncoding, Input>,
     ) -> Result<()> {
         const TOP_LEVEL_DEPTH: usize = 0;
+        use ExpandedStreamItem::*;
 
         let mut is_first_item = true;
         let mut has_printed_skip_message = false;
         loop {
-            let item = reader.next_item()?;
-            let is_last_item = matches!(item, SystemStreamItem::EndOfStream(_));
-            item.raw_stream_item()
-                .map(|i| self.confirm_encoding_is_supported(i.encoding()))
-                .unwrap_or(Ok(()))?;
-
```

Comment:
🗺️ Prior to version `rc.7`, `ion-rust` didn't have a way to get the next encoded item in the stream AND the values to which it expanded. You could only get one or the other. This meant that you couldn't display the encoding of an e-expression because the reader would only surface its expanded values. The code snippet that follows uses `reader.next_expanded_item()`, which returns an enum of either values (literal or ephemeral) or e-expressions.